app/browsered/input-confirm.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+module.exports = () => {
+  const inputs = document.querySelectorAll('[data-confirmation]')
+
+  inputs.forEach(input => {
+    input.addEventListener('input', update, false)
+  })
+
+  const addFor = (input, confirmation) => {
+    const formGroup = input.closest('.form-group')
+
+    formGroup.after(confirmation)
+  }
+
+  function update (e) {
```

Comment:
more meaningful parameters, perhaps?

Reply:
This was a copy-paste from selfservice and I believe the naming is meant to be very generic to support any form update, thus the term `update` rather than being specific. Might wanna double check with @maxcbc as he the original author of this.